sql/src/test/java/io/crate/integrationtests/UserDefinedFunctionsIntegrationTest.java

```diff
@@ -41,7 +42,7 @@
 
 import static org.hamcrest.CoreMatchers.is;
 
-@ESIntegTestCase.ClusterScope(numDataNodes = 2, numClientNodes = 0, randomDynamicTemplates = false)
+@ESIntegTestCase.ClusterScope(numDataNodes = 1, numClientNodes = 0, randomDynamicTemplates = false)
```

Comment:
why is that necessary? the tests should also run in a multi node environment. We already had a bug which occured only in a multi node env.